docs/dev/retool/workflow.md

```diff
@@ -1 +1,74 @@
 # Retool Workflow
+
+[Retool workflows](https://retool.com/products/workflows) are great for automated tasks that need to be run on intervals. For our project, we use Retool workflows to routinely sync our database with Legal Server every day. Additionally, we take advantage of Retool workflows to create function abstractions that chain REST APIs to create, edit, or delete listings (i.e. limited case assignments) via the Retool application.
+
+## Legal Server Sync
```

Comment:
should be "LegalServer", but also, thoughts on renaming this section to "LegalServer Sync Workflow"?